### DISCUSSION TITLE
tip for keyboard shortcut for "search" should be updated

### DISCUSSION THREAD

**Discussion by madbrain76** [home-assistant/frontend#12022] [Other feature requests] [answered]
The following tip sometimes shows up at the bottom of the "configuration" screen :
"tip! press 'c' on any page to open this search bar."

When the tip appear, the search bar is not actually being shown yet. Therefore, the statement is at best confusing, and at worst incorrect. IMO, the tip should be changed to :
"tip! press 'c' on any page to open the search bar."

**Comment by matthiasdebaat** [comment]
We're improving the Quick Bar by making it easier to find and turning it into a [single search bar](https://github.com/home-assistant/frontend/discussions/11511). Making it into a single search bar we have to streamline the tip and keyboard shortcut for example cmd + K on Mac.

**Comment by Mariusthvdb** [comment]
please allow me this here, I couldn't find another related topic in the discussions. I must, Ill post in a separate topic.

just noticed the new Pr https://github.com/home-assistant/frontend/pull/12355 in Frontend latest, introducing the Tip on Set State. Is there any rationale behind this? It feels really out of place on a developers tools page, not to mention it simply takes costly screen estate:

Unknown to any other open discussion on the matter, I ofc don't know if this is going to be implemented in other places. If so, please have some public polling on this, or at least allow it to be optional.

It coincides with the same frontend experience as the Magnifier we now have on All (!) our views, which might be nice for some, but really distracting for others. https://github.com/home-assistant/frontend/discussions/12384

<img width="570" alt="Schermafbeelding 2022-04-25 om 09 23 41" src="https://user-images.githubusercontent.com/33354141/165040219-69298bbc-f810-4d77-9566-9a797972e267.png">

especially on mobile:

![20220425_072526000_iOS](https://user-images.githubusercontent.com/33354141/165040617-7c97f73e-f24d-4ac9-a84e-d00354a59890.jpg)

**Reply by matthiasdebaat** [reply]
That's a different discussion, please open a new one.

**Reply by Mariusthvdb** [reply]
I made https://github.com/home-assistant/frontend/issues/12406, but its posted under Issues, could you please move it to Discussion?

**Comment by zsarnett** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/12458
